docs/cookbook/extending-product-list.md

```diff
@@ -84,23 +191,24 @@ use Shopsys\ReadModelBundle\Product\Listed\ListedProductView as BaseListedProduc
 class ListedProductViewFactory extends BaseListedProductViewFactory
 {
     /**
-     * @param \Shopsys\ShopBundle\Model\Product\Product $product
+     * @param array $productArray
      * @param \Shopsys\ReadModelBundle\Image\ImageView|null $imageView
      * @param \Shopsys\ReadModelBundle\Product\Action\ProductActionView $productActionView
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroup $pricingGroup
      * @return \Shopsys\ReadModelBundle\Product\Listed\ListedProductView
      */
-    public function createFromProduct(Product $product, ?ImageView $imageView, ProductActionView $productActionView): BaseListedProductView
```

Comment:
You really need to extend `createFromProduct` method too because it's used for example for top products which uses productListMacro too